Frame.cc

```diff
@@ -633,9 +639,10 @@ bool Frame::FillRegionHistogramData(int region_id, CARTA::RegionHistogramData* h
                             region->CalcHistogram(config_channel, curr_stokes, num_bins, min_val, max_val, data, *new_histogram);
                         }
                     } else {
+                        ChannelRange single_channel(config_channel);
```

Comment:
It seems unnecessarily verbose to create named variables for these -- I was suggesting anonymous objects, so `ChannelRange(some_channel_here)` or just `ChannelRange()`. But I don't feel that strongly about it, and it's only a few places in the code, so if you prefer this style I don't mind if you keep it.

Reply:
Agree. It looks cleaner using anonymous objects. I will change them and add a note on how to use ChannelRange structure.